pkg/reconciler/mongodb/mongodbsource_test.go

```diff
@@ -57,17 +95,52 @@ func TestAllCases(t *testing.T) {
 		Name: "key not found",
 		// Make sure Reconcile handles good keys that don't exist.
 		Key: "foo/not-found",
-	}}
-
-	// TODO add more UTs
+	}, {
+		Name:    "missing sink",
+		WantErr: true,
+		Objects: []runtime.Object{
+			NewMongoDbSourceV1Alpha1(sourceName, testNS,
```

Comment:
nit: we only have v1alpha1 here, so no need to explicitly mention it in the function name. Rename it to NewMongoDbSource

Reply:
Doen